common/elasticsearch/client/os2/client_bulk.go

```diff
@@ -80,6 +80,8 @@ func (v *bulkProcessor) Add(request *bulk.GenericBulkableAddRequest) {
 			return
 		}
 		req.Action = "index"
+		req.Version = &request.Version
+		req.VersionType = &request.VersionType
```

Comment:
let's add unit test to compare `req` against expected payload

Reply:
Added test